ext/uri/uriparser/include/uriparser/Uri.h

```diff
@@ -87,6 +87,10 @@ extern "C" {
 
 
 
+extern const URI_CHAR * const URI_FUNC(SafeToPointTo);
```

Comment:
If the `uriparser_copy_uri()` function can be added to uriparser then the following changes will be unnecessary